docs/serving/configuring-autoscaling.md

```diff
@@ -526,6 +526,43 @@ spec:
 {{< /tab >}}
 {{< /tabs >}}
 
+
+### Scale To Zero Last Pod Retention Period
+
+This flag ensures that after the decision has been made by the Autoscaler to scale to 0 the last pod will remain active for at least this amount of timei, i.e minimum amount of time the last pod will linger. This is in contrast to `scale-to-zero-grace-period` flag, that determines the maximum amount of time that the last pod will stay alive.
```

Comment:
The `scale-to-zero-grace` flag determines the **minimum** amount of time in seconds that the least pod will remain active after the Autoscaler has decided to scale pods to zero.

**NOTE:** This contrasts with the `scale-to-zero-grace-period` flag, which determines the **maximum** amount of time in seconds that the least pod will remain active after the Autoscaler has decided to scale pods to zero.

Reply:
Done.